README.md

```diff
@@ -54,8 +57,9 @@ Shaderc into.
 ```sh
 git clone https://github.com/google/shaderc $SOURCE_DIR
 cd $SOURCE_DIR/third_party
-git clone https://github.com/google/googletest.git
-git clone https://github.com/google/glslang glslang
+git clone https://github.com/google/googletest.git googletest
+git clone https://github.com/google/glslang.git glslang
+git clone https://github.com/KhronosGroup/SPIRV-Tools.git third_party/spirv-tools
```

Comment:
Good catch about `third_party`.

re: last argument: It's not necessary, but thus we have consistency and it looks better. But I removed it since there are concerns around it.